### DISCUSSION TITLE
RTL8821CU bluetooth not works, any advices?

### DISCUSSION THREAD

**Discussion by KarenSRG** [chris1111/Wireless-USB-Big-Sur-Adapter#318] [Q&A]
Thanks to dev, for the great work. Can anybody help me with my dongle(wifi/bt) with RTL8821CU to run bluetooth with this drivers. Bluetooth work's fine without them...

**Comment by KarenSRG** [comment]
My main goal is to run AirDrop is it real?
Specs:
QD4
Xeon 2667v4
Saphire 6600XT
OS: Ventura
OpenCore 0.9.8

**Reply by chris1111** [reply]
AirDrops will never work with this drivers
